runbotics/runbotics-orchestrator-ui/src/src-app/hooks/useProcessCollection.ts

```diff
@@ -55,13 +53,12 @@ const useProcessCollection = () => {
             .then(() => {
                 getCollectionPath();
             })
-            .catch(() => {});
+            .catch(() => { });
```

Comment:
could you do something with it? throw an error or display a toast whatever

Reply:
After merging two endpoints - first for fetching collections in parent collection and second for fetching all ancestors - this catch is no longer empty :)